osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/OsDAIConformanceCheckMetaTask.java

```diff
@@ -191,6 +191,11 @@ public void executeTransaction(EntityManager em) throws Exception {
                 ApplianceSoftwareVersion currentSoftwareVersion = ds.getVirtualSystem().getApplianceSoftwareVersion();
                 boolean doesSvaVersionMatchVsVersion = false;
                 for (OsImageReference imageRef : ds.getVirtualSystem().getOsImageReference()) {
+                	
+                	if (sva.getImage() == null) {
```

Comment:
I pushed changes relating to another bug. Inside that loop, I have seen the getImage to return null and this is the only sensible way to deal with the situation.